### PR TITLE
test(mountinapp, multiactionbutton,navigationbar): add cyress tests FE-2353

### PR DESCRIPTION
**Pull request by KatarzynaQA**
### Proposed behaviour

add cypress tests for multiActionButton default story and remove classic story, minor refactor for
mountInApp and NavigationBar components

<del> - [ ] Release notes (Conventional Commits)  <del>
<del>  - [ ] Unit tests <del>  
- [x] Cypress automation tests

<del>  - [ ] Storybook added or updated <del>  
<del>  - [ ] Theme support <del>  
<del>  - [ ] Typescript `d.ts` file added or updated <del>  

### Testing instructions

run: npm run test-cypress and choose proper feature file

### CODE REVIEW

File: cypress/features/regression/multiActionButton.feature [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+      | 1234567890              |
+      | áéíóú¿¡üñ               |
+      | !@#$%^*()_+-=~[];:.,?{} |
+      | ÄÖÜßäöüß                |
+  # @ignore because of FE-1447
+  # | <>                       |
+
+  @positive
+  Scenario: Invoking Multi Action Button component
+  When I hoverover on Multi Action Button
```
Comment (DlgSHi) [inline cypress/features/regression/multiActionButton.feature]: indent

File: cypress/support/step-definitions/multi-action-button-steps.js [hunk 2]
```diff
@@ -1,65 +1,58 @@
...
-  multiActionButtonPreview().trigger('mouseover');
+Then('Multi Action Button subtext on preview is {string}', (subtext) => {
+  multiActionButtonSubtext().should('have.text', subtext);
+});
+
+Then('Multi Action Button has golden border color', () => {
+  cy.focused().should('have.css', 'border', 'rgb(255, 181, 0) solid 3px');
+});
+
+When('I hoverover on Multi Action Button', () => {
```
Comment (DlgSHi) [inline cypress/support/step-definitions/multi-action-button-steps.js]:
```suggestion
When('I hover on Multi Action Button', () => {
```

File: cypress/locators/multi-action-button/locators.js [hunk 3]
```diff
@@ -1,3 +1,5 @@
 // component preview locators
-export const MULTI_ACTION_BUTTON_PREVIEW = 'div[data-component="multi-action-button"]';
-export const MULTI_ACTION_BUTTON_LIST = 'div[data-element="additional-buttons"]';
+export const MULTI_ACTION_BUTTON_COMPONENT = '[data-component="multi-action-button"]';
+export const MULTI_ACTION_BUTTON_LIST = '[data-element="additional-buttons"]';
+export const MULTI_ACTION_BUTTON_SUBTEXT = '[data-element="subtext"]';
```
Comment (DlgSHi) [inline cypress/locators/multi-action-button/locators.js]: this locator is already defined in `./cypress/locators/button/locators.js`

File: cypress/locators/multi-action-button/locators.js [hunk 4]
```diff
@@ -1,3 +1,5 @@
...
+export const MULTI_ACTION_BUTTON_TEXT = '[data-element="main-text"]';
```
Comment (DlgSHi) [inline cypress/locators/multi-action-button/locators.js]: and this one is already defined in `./carbon/cypress/locators/pages/locators.js`

File: cypress/support/step-definitions/multi-action-button-steps.js [hunk 5]
```diff
@@ -1,65 +1,58 @@
...
-    .children().children().first()
-    .should('have.text', text);
+  multiActionButtonText().should('have.text', text);
 });
 
 Then('Multi Action Button state is disabled', () => {
-  multiActionButtonPreview()
-    .children()
-    .should('have.attr', 'disabled');
+  multiActionButton().should('have.attr', 'disabled');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/multi-action-button-steps.js]: `.and('be.disabled')`?

File: cypress/support/step-definitions/multi-action-button-steps.js [hunk 6]
```diff
@@ -1,65 +1,58 @@
...
 });
 
-When('I invoke Multi Action Button component', () => {
-  multiActionButtonPreview().trigger('mouseover');
+Then('Multi Action Button subtext on preview is {string}', (subtext) => {
+  multiActionButtonSubtext().should('have.text', subtext);
+});
+
+Then('Multi Action Button has golden border color', () => {
+  cy.focused().should('have.css', 'border', 'rgb(255, 181, 0) solid 3px');
```
Comment (DlgSHi) [inline cypress/support/step-definitions/multi-action-button-steps.js]: what about `cy.focused()` ?

Comment (KatarzynaQA) [reply cypress/support/step-definitions/multi-action-button-steps.js]: fixed

File: cypress/locators/multi-action-button/locators.js (hunk 4 above)
Comment (KatarzynaQA) [reply cypress/locators/multi-action-button/locators.js]: fixed

File: cypress/locators/multi-action-button/locators.js (hunk 3 above)
Comment (KatarzynaQA) [reply cypress/locators/multi-action-button/locators.js]: fixed

File: cypress/support/step-definitions/multi-action-button-steps.js (hunk 2 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/multi-action-button-steps.js]: fixed

File: cypress/features/regression/multiActionButton.feature (hunk 1 above)
Comment (KatarzynaQA) [reply cypress/features/regression/multiActionButton.feature]: fixed

File: cypress/support/step-definitions/multi-action-button-steps.js (hunk 5 above)
Comment (KatarzynaQA) [reply cypress/support/step-definitions/multi-action-button-steps.js]: it doesn't work here

**Pull request merged**